pkgs/development/libraries/raylib/default.nix

```diff
@@ -3,25 +3,26 @@
   libX11, libXi, libXcursor, libXrandr, libXinerama,
   alsaSupport ? stdenv.hostPlatform.isLinux, alsaLib,
   pulseSupport ? stdenv.hostPlatform.isLinux, libpulseaudio,
+  sharedLib ? true,
   includeEverything ? true
 }:
 
 stdenv.mkDerivation rec {
   pname = "raylib";
-  version = "3.5.0";
+  version = "3.7.0";
 
   src = fetchFromGitHub {
     owner = "raysan5";
     repo = pname;
     rev = version;
-    sha256 = "0syvd5js1lbx3g4cddwwncqg95l6hb3fdz5nsh5pqy7fr6v84kwj";
+    sha256 = "1w8v747hqy0ils6zmy8sm056f24ybjhn9bamqzlxvbqhvh9vvly1";
   };
 
   patches = [
-    # fixes examples not compiling in 3.5.0
+    # fixes incorrect version being set by cmake
     (fetchpatch {
-      url = "https://patch-diff.githubusercontent.com/raw/raysan5/raylib/pull/1470.patch";
-      sha256 = "1ff5l839wl8dxwrs2bwky7kqa8kk9qmsflg31sk5vbil68dzbzg0";
+      url = "https://patch-diff.githubusercontent.com/raw/raysan5/raylib/pull/1761.patch";
```

Comment:
```suggestion
      url = "https://github.com/raysan5/raylib/commit/204aa4c46fdd6986aa0130eeba658562c540759f.patch";
```

Reply:
The contents of these two files are 1:1
https://patch-diff.githubusercontent.com/raw/raysan5/raylib/pull/1761.patch
https://github.com/raysan5/raylib/commit/204aa4c46fdd6986aa0130eeba658562c540759f.patch

I'm curious what this change does?

Reply:
I didn't check the PR but if it would be unmerged we could get hash mismatches if it changes and it is nicer to fetch a commit since it is easier to track that down later on.